src/shared/store/actions/project.ts

```diff
@@ -1,9 +1,13 @@
 import { Action, ActionCreator, Dispatch } from 'redux';
-import { Project } from '@bbp/nexus-sdk';
+import { Project, ElasticSearchView } from '@bbp/nexus-sdk';
 import { ThunkAction } from '..';
 import { CreateProjectPayload } from '@bbp/nexus-sdk/lib/Project/types';
 import { httpGet, httpPut } from '@bbp/nexus-sdk/lib/utils/http';
 import { IdentityResponse, Identity } from '@bbp/nexus-sdk/lib/ACL/types';
+import { asyncTimeout } from '../../utils';
+import { notification } from 'antd';
+
+const DEFAULT_ELASTIC_SEARCH_INDEX_ID = 'nxv:defaultElasticSearchIndex';
```

Comment:
Don't we have this elsewhere already? Or as part of the SDK?

Reply:
You're right, it's actually not necessary to specify an ID if you just want the default view. Neato!